From 2a845fdf26149b1fc58eaefa3841a7ecfec7acef Mon Sep 17 00:00:00 2001
From: "zongren.yin" <zongren.yin@amlogic.com>
Date: Wed, 13 Dec 2023 09:41:46 +0800
Subject: [PATCH] amlbian: S4 no convert plugin [1/1]

PD#SWPL-149729

Problem:
Copy video convert plugin failed

Solution:
Remove video convert copying

Fixed conflicts:
NA

Verify:
local

Signed-off-by: zongren.yin <zongren.yin@amlogic.com>
Change-Id: I09307835a6dfc931d934866db0f87603a1aaba2b
---
 env/copy_debs_from_yocto.sh | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/env/copy_debs_from_yocto.sh b/env/copy_debs_from_yocto.sh
index dfe090f..7eb20c3 100755
--- a/env/copy_debs_from_yocto.sh
+++ b/env/copy_debs_from_yocto.sh
@@ -113,7 +113,7 @@ packages_arr=(  procrank readline ncurses \
                 aml-audio-service aml-avsync aml-cas-hal aml-dtvdemod aml-dvb \
                 aml-dvbaudioutils aml-hdcp aml-hdcp-load-firmware aml-libdvr \
                 aml-mediahal-sdk aml-pqserver gst-plugin-aml-v4l2dec\
-                libge2d gst-aml-gfx2d gst-aml-dma-allocator dma-allocator gst-plugin-conv \
+                libge2d gst-aml-gfx2d gst-aml-dma-allocator dma-allocator \
                 aml-provision aml-ubootenv aml-zvbi optee-video-firmware\
                 android-tools openssl android-tools-logcat bluez-alsa bluez5 \
                 fbscripts gst-aml-drmbufferpool-plugins \
@@ -179,7 +179,7 @@ KERNEL_DEBS_DIR=$BUILDDIR/tmp/work/$TARGET_BOARD
 case "$TARGET_BOARD" in
 	mesont7_an400_debian_5.15-poky-linux)	# T7/A311D2
 		packages_arr+=( libamvenc libmultienc gst-plugin-venc-amlvenc adla-lib \
-                        libjpeg gst-plugin-venc-jpeg \
+                        libjpeg gst-plugin-venc-jpeg gst-plugin-conv \
                         npu-common detect-library gst-plugin-nn gst-plugin-overlay \
                         libgles-eabihf-gondul-wayland-drm \
                         mediactrlsrc \
@@ -190,7 +190,7 @@ case "$TARGET_BOARD" in
 		;;
 	mesong12b_w400_debian_5.15-poky-linux) # A311D
 		packages_arr+=( libamvenc libamvenc-264 libamvenc-265 gst-plugin-venc-amlvenc \
-                        libjpeg gst-plugin-venc-jpeg \
+                        libjpeg gst-plugin-venc-jpeg gst-plugin-conv \
                         npu-common detect-library gst-plugin-nn gst-plugin-overlay \
                         h264bitstream \
                         libgles-eabihf-gondul-wayland-drm \
@@ -200,7 +200,7 @@ case "$TARGET_BOARD" in
 		;;
 	mesonsm1_ac200_debian_5.15-poky-linux) # S905D3
         	packages_arr+=( libamvenc libamvenc-264 libamvenc-265 h264bitstream gst-plugin-venc-amlvenc \
-                            libjpeg gst-plugin-venc-jpeg \
+                            libjpeg gst-plugin-venc-jpeg gst-plugin-conv \
                             npu-common detect-library gst-plugin-nn gst-plugin-overlay \
                             h264bitstream \
                             libgles-eabihf-dvalin-wayland-drm \
@@ -216,7 +216,7 @@ case "$TARGET_BOARD" in
 		;;
 	mesonsc2_ah212_debian_5.15-poky-linux) # S905X4
         	packages_arr+=( libamvenc libamvenc-264 libamvenc-265 gst-plugin-venc-amlvenc \
-                            libjpeg gst-plugin-venc-jpeg \
+                            libjpeg gst-plugin-venc-jpeg gst-plugin-conv \
                             h264bitstream \
                             libgles-eabihf-dvalin-wayland-drm \
                             mediactrlsrc \
-- 
2.25.1

